tests/test_package_toml.py

```diff
@@ -14,6 +16,24 @@ def test_get_build_system_from_pyproject_toml_unknown():
     )
 
 
+def test_build_manager_sucess_command():
+    bpm = BuildPlanManager(args=Mock())
+    try:
+        bpm.execute(build_plan=[["sh", "/tmp", "pwd"]], zip_stream=None, query=None)
+    except RuntimeError:
```

Comment:
There's no need to catch such errors in tests, any exceptions wil fail the test.